static/app/components/dashboards/issueWidgetQueriesForm.tsx

```diff
@@ -50,6 +54,19 @@ class IssueWidgetQueriesForm extends React.Component<Props, State> {
     };
   };
 
+  tagValueLoader = (key: string, search: string) => {
+    const {organization, selection} = this.props;
+    const orgId = organization.slug;
+    const projectIds = selection.projects.map(id => id.toString());
+    const endpointParams = {
+      start: getUtcDateString(selection.datetime.start),
+      end: getUtcDateString(selection.datetime.end),
+      statsPeriod: selection.datetime.period,
+    };
+
+    return fetchTagValues(this.props.api, orgId, key, search, projectIds, endpointParams);
```

Comment:
nit: retrieve `api` when destructuring other props in line 58

Reply:
Fixed!